### ISSUE TITLE
your documentation on 'save' only shows 2 parameters missing userAuthenticationRequired

### ISSUE THREAD

**Issue by sfitzjava** [sjhoeksma/cordova-plugin-keychain-touch-id#40]
Hi, 
  Looks like ionic's native wrapper read the Readme.md information which does not show userAuthenticationRequired as one of the parameters to the save() function.

  This causes their wrapper to fail.   If you could update the docs to show all needed parameters then maybe ionics will realize their mistake and fix the issue.

Thanks,

**Comment by Muzietto** [comment]
I agree the documentation is confusing. In any case, I've filed another issue because my experience tells that userAuthenticationRequired is no longer in place. Just ignore it.
